x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/form/deserializer_and_serializer.test.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { setAutoFreeze } from 'immer';
+import { cloneDeep } from 'lodash';
```

Comment:
Shouldn't we use the `lodash.clonedeep` package?

Reply:
I think not after https://github.com/elastic/kibana/pull/78156 was merged. We should be importing the entire lodash library now.

Reply:
Could. Thanks, good to know 👍